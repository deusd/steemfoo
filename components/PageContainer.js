import React from 'react'
import { Container, Content } from 'native-base'

const PageContainer = (props: any) => (
  <Container {...props}>
    <Content>{props.children}</Content>
  </Container>
)

export default PageContainer
