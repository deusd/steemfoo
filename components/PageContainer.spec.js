import 'react-native'
import { shallow } from 'enzyme'
import React from 'react'
import { View } from 'react-native'
import PageContainer from './PageContainer'

describe('PageContainer component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<PageContainer />)
  })

  it('should render', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should render with children', () => {
    wrapper.setProps({ children: <View /> })
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.contains(<View />)).toBe(true)
  })
})
