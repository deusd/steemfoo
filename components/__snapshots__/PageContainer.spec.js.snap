// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContainer component should render 1`] = `
<Styled(Container)>
  <Styled(Content) />
</Styled(Container)>
`;

exports[`PageContainer component should render with children 1`] = `
<Styled(Container)>
  <Styled(Content)>
    <ForwardRef(View) />
  </Styled(Content)>
</Styled(Container)>
`;
